docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -579,6 +579,28 @@ const classes = makeStyles(theme => ({
   />
   ```
 
+### Popper
+
+- Upgrade [Popper.js](https://github.com/popperjs/popper-core) from v1 to v2.
+  This third-party library has introduced a lot of changes.<br />
+  You can read [their migration guide](https://popper.js.org/docs/v2/migration-guide/) or the following summary:
+
+  - The CSS prefixes have changed:
+    ```diff
+    popper: {
+      zIndex: 1,
+    - '&[x-placement*="bottom"] $arrow': {
+    + '&[data-popper-placement*="bottom"] $arrow': {
+    ```
+  - Method names have changed.
+    ```diff
+    -popperRef.current.scheduleUpdate()
+    -popperRef.current.update()
+    +popperRef.current.update()
+    +popperRef.current.forceUpdate()
```

Comment:
```suggestion
    -popperRef.current.update()
    +popperRef.current.forceUpdate()

    -popperRef.current.scheduleUpdate()
    +popperRef.current.update()
```

It's otherwise slightly disorienting what new name belongs to what old name

Reply:
I think that we would need two different ```diff blocks in this case, otherwise would great the default behavior of git diff.